projects/components/src/multi-select/multi-select.component.test.ts

```diff
@@ -267,18 +310,19 @@ describe('Multi Select Component', () => {
       }
     );
 
-    spectator.tick();
-    expect(spectator.query('.search-bar')).toExist();
-    spectator.click('.search-bar');
+    spectator.click('.trigger-content');
+
+    const searchBar = spectator.query('.search-bar', { root: true });
+    expect(searchBar).toExist();
 
-    spectator.triggerEventHandler(SearchBoxComponent, 'valueChange', 'fi');
```

Comment:
Not sure how these tests worked before. PopoverModule was not added in the shallow test. So the popover content could never get projected. 

`triggerEventHandler` doesn't provide a `{root: true}` option. Since the search box would be outside the host element's scope, this trigger call just doesn't work.

Reply:
> Not sure how these tests worked before. PopoverModule was not added in the shallow test.

That's how - it wasn't using the real popover (so those tags are basically ignored), which your test is now. So only the searchbox was mocked as an actual component - but it was never projected. That's also why the root wasn't there before.

Reply:
You also might be able to use `read: DebugElement` to trigger the event handler since you can't use a selector directly in there.